packages/workers/package.json

```diff
@@ -15,13 +15,17 @@
         "clean": "rimraf dist *.tsbuildinfo",
         "build": "tsc",
         "worker:alerts:tg": "ts-node src/scripts/il-alerts.ts",
-        "worker:alrets:discord": "ts-node src/scripts/il-alerts-discord.ts"
+        "worker:alrets:discord": "ts-node src/scripts/il-alerts-discord.ts",
```

Comment:
I misspelled `alerts` here, can you fix it?

Reply:
yes

Reply:
😂